website/util/sanitize.py

```diff
@@ -11,6 +11,10 @@ def strip_html(unclean):
     :return: stripped string
     :rtype: str
     """
+    # Handle non string inputs so this function can be used with higher-order
+    # functions, such as rapply (recursively applies a function to collections)
+    if not isinstance(unclean, basestring):
+        return unclean
```

Comment:
As discussed in flowdock based on use case, maybe change this to `if isinstance(unclean, bool)` instead (to make the exemption from sanitizer as specific as possible, etc).